### ISSUE TITLE
Upload to specific folder

### ISSUE THREAD

**Issue by SteveCurran** [Microsoft/vscode-azure-account#72]
When I upload to the cloudshell  using this extension it  is placed in the cloud storage disk image. However, when I try to execute bash scripts from here I receive a permission denied error.  If I manually upload the bash script to the fileshare and not the disk image I am able to execute the bash script from the bash window like this:
./clouddrive/script.sh

Maybe I am confused, however, is there a way to upload to the fileshare instead of  the disk image?

**Comment by chrmarti** [comment]
This is consistent with `shell.azure.com`. You can set the executable bit with `chmod +x script.sh`.

**Comment by SteveCurran** [comment]
@chrmarti Thanks for the execution tip. However, I am looking at the last comments on issue #34 and it states that powershell supports uploading to the clouddrive folder and that bash currently does not. Maybe a feature request to allow a user setting to upload to a certain folder in cloud storage?

**Comment by chrmarti** [comment]
Indeed, tracking as feature request. This will require support from the CloudShell first before we can add it to the VS Code extension.
